components/Seekbar.tsx

```diff
@@ -1,34 +1,34 @@
-import React, { type FunctionComponent, useContext } from 'react';
+import React, { type FunctionComponent, useContext   } from 'react';
 import { AudioPlayerContext } from '../contexts/audio_player';
-import { Slider } from 'react-native-elements';
+import { Slider } from '@rneui/themed';
 import { View, Text } from 'react-native';
 
 interface OwnProps {
   currTime: number;
-    className?: string;
+  className?: string;
 }
 
 type Props = OwnProps;
 
 const Seekbar: FunctionComponent<Props> = (props) => {
   // const [valueSlider, setValueSlider] = useState(0);
-
   const audioContext = useContext(AudioPlayerContext);
 
-
     return (
       <View className={"w-full flex justify-center"}>
        <Slider
         value={audioContext.audioState.currentTrack ? props.currTime / audioContext.audioState.currentTrack.track_duration * 100 : 0}
```

Comment:
Nog een klein dingetje dat me opviel is dat de seconden niet van 0 tot 59 is, maar van 0 tot 99, klein detail, maar wel nice om dat goed te hebben denk ik.